### DISCUSSION TITLE
Paperless-ngx Update doesn't work

### DISCUSSION THREAD

**Discussion by dodi666** [tteck/Proxmox#3622] [Help Needed]
Hi,
yesterday i tried to update Paperels-ngx via "update" command in lxc.
The update process stops with error message (see below)
Any ideas?

Thanks for help.
Sascha

![grafik](https://github.com/user-attachments/assets/fa75f353-547f-4f31-8b05-88a50f01e905)

**Comment by tteck** [comment]
Search Discussions and issues before asking.

https://github.com/tteck/Proxmox/issues?q=paperless
